TomTomSDKExamples/Examples/UseCase/OnlineNavigationContent.swift

```diff
@@ -14,8 +14,6 @@ import Foundation
 import SwiftUI
 
 // TomTomSDK modules
-import TomTomSDKCommon
```

Comment:
These libraries are mentioned in the "[Building a navigation app](https://developer.tomtom.com/ios/navigation/documentation/use-cases/building-a-navigation-app)" use case in the developer portal.

Reply:
On the portal they will be removed as part of the subsequent release. This is already internally addressed.